### ISSUE TITLE
Send Text from RQuickShare to Android

### ISSUE THREAD

**Issue by rodrigoswz** [Martichou/rquickshare#67]
Receiving text from Android works perfectly, it would be great if it also worked the other way around.

**Comment by Martichou** [comment]
I agree, but I have a bit of trouble to figure out how this can be possible in a UX friendly way.
The best of the best would be to have something added in the right click contextual menu, but I don't think that's possible.

Anything other than that would require to copy the text on the computer, open RQuickShare, enter special mode in RQuickShare, paste the text, send it to the desired device.

I'm not sure about all of this, so any idea is welcome!

**Comment by bryanniwa** [comment]
Apologies for any ignorance but the windows version works if you just open the QuickShare application and press CTRL+V. Is there any reason it couldn't be implemented in a similar fashion here? I'd be interested in taking this on if it's something you'd want added.

**Comment by Martichou** [comment]
Hey! No there no reason.. I just never thought about CTRL+V to send text :) 
I'll reopen this issue to track that, will do when I have some time 😃

**Comment by HannesGitH** [comment]
(as im now a little bit more familiar with rquickshare) I'd like to tackle this, if you want, you can assign me :)
I'll probably try tomorrow morning while in transit

**Comment by kaisar945** [comment]
We could add an option to automatically sync the clipboard, which is a perfect solution. What do you think?

**Comment by nunocruz19** [comment]
> Receiving text from Android works perfectly, it would be great if it also worked the other way around.

+1

Using Ctrl+V would be perfect 👌

**Comment by Anthonyy232** [comment]
Windows Link also does clipboard automatically, so while native quick share doesn't have this, it would really be great to have an equiv. feature on mac.

**Comment by siddharthalodha** [comment]
+1 this seems like a great feature to support.

**Comment by AuthenticSm1les** [comment]
+1
